README.md

```diff
@@ -33,183 +144,227 @@ You can also tab complete remote names and branch names e.g.: `git pull or<tab>
-* ABCD represent the index; ` | ` (`DelimText`); EFGH represent the working directory
- * `+` = Added files
- * `~` = Modified files
- * `-` = Removed files
- * `!` = Conflicted files
- * As in `git status`, index status is dark green and working directory status is dark red
-
-* W represents the overall status of the working directory
- * `!` = There are unstaged changes in the working tree (`LocalWorkingStatus`)
- * `~` = There are uncommitted changes i.e. staged changes in the working tree waiting to be committed (`LocalStagedStatus`)
- * None = There are no unstaged or uncommitted changes to the working tree (`LocalDefault`)
-* `]` (`AfterText`)
+```powershell
+# my profile.ps1
+function prompt {
+    # Your non-prompt logic here
 
-The symbols and surrounding text can be customized by the corresponding properties on `$GitPromptSettings`.
+    # Have posh-git display its default prompt
+    & $GitPromptScriptBlock
+}
+```
 
-For example, a status of `[master ≡ +0 ~2 -1 | +1 ~1 -0]` corresponds to the following `git status`:
+And if you'd like to write prompt text before and/or after the posh-git default prompt:
 
 ```powershell
-# On branch master
-#
-# Changes to be committed:
-#   (use "git reset HEAD <file>..." to unstage)
-#
-#        modified:   this-changed.txt
-#        modified:   this-too.txt
-#        deleted:    gone.ps1
-#
-# Changed but not updated:
-#   (use "git add <file>..." to update what will be committed)
-#   (use "git checkout -- <file>..." to discard changes in working directory)
-#
-#        modified:   not-staged.ps1
-#
-# Untracked files:
-#   (use "git add <file>..." to include in what will be committed)
-#
-#        new.file
+# my profile.ps1
+function prompt {
+    # Your non-prompt logic here
+    $prompt = Write-Prompt "Text before posh-git prompt" -ForegroundColor Orange
+    $prompt += & $GitPromptScriptBlock
+    $prompt += Write-Prompt "Text after posh-git prompt" -ForegroundColor ([ConsoleColor]::Magenta)
+    if ($prompt) { $prompt } else { " " } # Prevents PowerShell from appending "PS>"
+}
 ```
 
-## Based on work by:
+## Based on work by
+
+- Keith Dahlby, http://solutionizing.net/
+- Mark Embling, http://www.markembling.info/
+- Jeremy Skinner, http://www.jeremyskinner.co.uk/
+
+[av-develop-site]: https://ci.appveyor.com/project/dahlbyk/posh-git/branch/develop
+[av-develop-img]: https://ci.appveyor.com/api/projects/status/eb8erd5afaa01w80/branch/develop?svg=true&pendingText=develop%20%E2%80%A3%20pending&failingText=develop%20%E2%80%A3%20failing&passingText=develop%20%E2%80%A3%20passing
+[av-master-site]: https://ci.appveyor.com/project/dahlbyk/posh-git/branch/master
+[av-master-img]: https://ci.appveyor.com/api/projects/status/eb8erd5afaa01w80/branch/master?svg=true&pendingText=master%20%E2%80%A3%20pending&failingText=master%20%E2%80%A3%20failing&passingText=master%20%E2%80%A3%20passing
+
+[tv-develop-img]: https://travis-ci.org/dahlbyk/posh-git.svg?branch=develop
```

Comment:
This link doesn't seem to be giving an updated/correct image.  Image indicates failure but when you click on it, the page indicates success.

Reply:
Curiously, Travis doesn't seem to support branch-specific links. https://github.com/travis-ci/travis-ci/issues/5024

I'm expecting this link will be correct once v1 becomes the default branch here.